UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Services/BuilderService.cs

```diff
@@ -291,5 +300,47 @@ private void LogProductVersions(ProductVersionsRequest productVersionsRequest, s
                 throw new Exception($"Logging Product version failed at {DateTime.Now.ToUniversalTime()} | _X-Correlation-ID:{CommonHelper.CorrelationID}", ex);
             }
         }
+
+        private async Task PerformAncillaryFilesOperationsAsync(string batchId, string exchangeSetPath, string correlationId, string readMeSearchFilter)
+        {
+            string exchangeSetRootPath = Path.Combine(exchangeSetPath, fssApiConfig.Value.EncRoot);
+            string readMeFilePath = Path.Combine(exchangeSetRootPath, fssApiConfig.Value.ReadMeFileName);
+            if (readMeSearchFilter == ReadMeSearchFilter.AVCS.ToString())
+            {
+                return;
+            }
```

Comment:
```suggestion

```

Reply:
Applied changes as suggested by Nadeem as this method contains ancillary files changes not only readme file related changes.